src/main/resources/com/rultor/agents/docker/image/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:14.04
+MAINTAINER Armin Braun <me@obrown.io>
+LABEL Description="This is the image holding the Rultor build runner infrastructure."
+
+# UTF-8 locale
+RUN locale-gen en_US en_US.UTF-8
+RUN dpkg-reconfigure locales
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+
+# Basics
```

Comment:
@mkordas we need it to get the `col` command, very basic *nix stuff imo. The current Rultor implementation blindly assumes it to be available on any host it SSHs into even :) You really need a very minimal install to not have it available imo.